### PR TITLE
feat: used two icons per organizer for socials, and standardized footer images

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Switches organizers to have two image icons below their names for their most emphasized social media / website links.

Also applies a few other touchups:

* Fixes organizer sorting by late by swapping "Gibson" and "Goldberg"
* Removes placeholder _"INSERT TEXT HERE YIPPEE"_ text from the Twitter share
* Gets the footer images to be all about the same. It was bugging me that two were differently sized 😄 

Screenshot of organizers and footers in the PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,7 @@ <h1 class="heading-primary">BarCamp 14</h1>
             </a>
             <a
               class="button share-link"
-              href="https://twitter.com/intent/tweet?url=http%3A%2F%2F2023.barcampphilly.org&text=INSERT%20TEXT%20HERE%20YIPPEE&hashtags=apple%2Cbanana"
+              href="https://twitter.com/intent/tweet?url=http%3A%2F%2F2023.barcampphilly.org&hashtags=apple%2Cbanana"
```
Comment (FlipperPA) [inline index.html:117]: What's up with the `apple banana` hashtag?

**Comment by FlipperPA** [comment]
Looks good to me @JoshuaKGoldberg - just the one comment. THANK YOU!

File: index.html (hunk 1 above)
Comment (JoshuaKGoldberg) [reply index.html:117]: Ah that was a placeholder one too, forgot about that 😄. will remove...!

Comment (JoshuaKGoldberg) [reply index.html:117]: 602e0f7..d8c56c2

**Pull request merged**